lib/ownership_client.rb

```diff
@@ -0,0 +1,20 @@
+require 'faraday'
+
+module OwnershipClient
+
+  def post(params)
+    conn = Faraday.new(:url => Rails.application.config.ownership_url)
+    conn.proxy Rails.application.config.ownership_url_default_proxy
+    conn.post '/ownerships', { :ownership => params }
```

Comment:
As the connection and proxy settings are going to be shared by a lot of the methods of the client maybe you can refactor them in its own method